src/js/components/Form/__tests__/Form-test-controlled.js

```diff
@@ -51,6 +52,30 @@ describe('Form controlled', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('custom theme', () => {
+    const customTheme = {
+      formField: {
+        feedback: {
```

Comment:
Can we use "survey" here, to align with using it elsewhere?

Reply:
yes